docs/getting_started/online_comp_getting_started.md

```diff
@@ -0,0 +1,84 @@
+#Getting Started on the Online Compiler
+
+The mbed Online Compiler delivers a full online editor, standard keyboard shortcuts, flexible workspace management and version control optimized for the mbed ecosystem.
+
+This article reviews the basics of interacting and creating and maintaining code in your workspace.
+
```

Comment:
Note: I deleted the video that was here because it was outdated. Do we have an updated one?